src/endpoints/http/collections/export/lib/parser.js

```diff
@@ -0,0 +1,40 @@
+import stream from 'stream';
+import jcsv from 'jcsv';
+import concat from 'concat-stream';
+import str from 'string-to-stream';
+
+class json extends stream.Transform {
+  constructor() {
+    super({ objectMode: true });
+  }
+
+  _transform(data, encoding, cb) {
+    data = JSON.stringify(data);
+    this.push(data);
+    cb();
+  }
+}
+
+export default {
+  bson: collection => Promise.resolve(collection),
+  json: collection => {
+    const parserJson = new json();
+    parserJson.filename = collection.filename;
+    return Promise.resolve(collection.pipe(parserJson));
+  },
+  csv: collection => {
+    const promise = new Promise((resolve, reject) => {
+      collection.pipe(concat(jsonArray => {
+        jcsv(jsonArray, { separator: ',', newline: "\r\n", headers: true }, (err, csv) => {
```

Comment:
@sedroche In many cases the records will have the same attributes, so I think using the first row to write headers should be enough. I don't think there is a good way to get all the headers of all the records without go through all of them.

Perhaps we can add warning message somewhere and explain that there might be fields missing in the header